update-changelog/index.ts

```diff
@@ -119,6 +121,29 @@ class UpdateChangelog extends Action {
 				labels: ['backport ' + versionMajorBranch, 'no-changelog'],
 			})
 		}
+
+		// publish changelog to community.grafana.com
+		const forumData = {
+			title: `Changelog: Updates in Grafana ${version}`,
+			raw: `${changelog}`,
+			category: 9,
+		}
+		const apiKey = getInput('grafanabotForumKey')
+
+		if (apiKey) {
+			axios({
```

Comment:
nit: since we usually `await`, I would rather `await` with `try/catch` on this `axios` call too.

Reply:
ok, wrapped in try/catch and awaiting the axios call